### PR TITLE
Add Debian packaging for the service

### PR DESCRIPTION
**Pull request by GabrielNagy**
I've built both the source and binary package, ensured there are no lintian warnings, tested the package locally, built it on Launchpad for all supported architectures, and ran amd64 autopkgtests locally.

DEENG-566

There are more minor fixes and changes detailed in the individual commits.

### CODE REVIEW

File: debian/copyright [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+Copyright: 2013 Patrick Mezard
+License: BSD-3
+
+Files: vendor/github.com/sirupsen/logrus/*
+Copyright: 2014 Simon Eskildsen
+License: MIT
+
+Files: vendor/github.com/sirupsen/logrus/alt_exit.go
+Copyright: 2012 Miki Tebeka <miki.tebeka@gmail.com>.
+License: MIT
```
Comment (denisonbarbosa) [inline debian/copyright:41]: Really, one file to break our globing? Ouch...

**Review by denisonbarbosa — APPROVED** [review]
I'm not an ace in packaging, so I could (and probably) have missed something. But everything looks great to me!

File: debian/control [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+               golang-go (>= 2:1.20~),
+               dbus,
+Standards-Version: 4.6.2
+XS-Go-Import-Path: github.com/ubuntu/ubuntu-proxy-manager
+Homepage: https://github.com/ubuntu/ubuntu-proxy-manager
+Description: Ubuntu Proxy Manager service
+ Ubuntu Proxy Manager is a D-Bus mediated service that allows for managing
+ system proxy settings via multiple backends (APT, environment variables and
+ GSettings).
+
```
Comment (didrocks) [inline debian/control:16]: Only one line break should be enough :)

**Review by didrocks — COMMENTED** [review]
Looks good to me! One nitpick for me and one thing to double check :)
Once addressed, feel free to merge, well done!

**Review by didrocks — APPROVED** [review]
Excellent way to end the week :)

**Pull request merged**